pkgs/by-name/tt/tt/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "tt";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "lemnos";
+    repo = "tt";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-oe0gegMINVmbw1UPvSy9G67D4amQTumlrgvb8H+Y4IM=";
+  };
+
+  vendorHash = "sha256-iny0OKInHqoXzYcEtd4f0T/yty5/z3k3yFbDo5yazes=";
+
+  postInstall = ''
+    mv $out/bin/src $out/bin/tt
+  '';
+
+  meta = with lib; {
+    description = "Typing test in the terminal written in Go";
+    homepage = "https://github.com/lemnos/tt";
+    license = licenses.mit;
+    maintainers = with maintainers; [ vinetos ];
+    platforms = platforms.linux;
```

Comment:
```suggestion
```
It seems to support macos as well.
https://github.com/lemnos/tt?tab=readme-ov-file#osx

Reply:
Excellent suggestion, happy to help test on `aarch64-darwin` 🍎

Reply:
yes but as I cannot test it I thought to let only linux 😅

Reply:
@Sail0rd, the GitHub tests (see checks below) will test building the package on macOS for you. In case there be any issues with building on a platform you are unable to test on, feel free to ping folks here or over on discourse and see if any one can help out 🙂

Reply:
Happy to report that I was able to build and run `tt` locally on `aarch64-darwin`, so please update the platforms to include `darwin` 🙂 

Please also consider removing `with lib;` from `meta =` as there is a preference to avoid it (for details see #208242).